packages/atomic/src/utils/jsx-utils.tsx

```diff
@@ -0,0 +1,44 @@
+import {Fragment, FunctionalComponent, h, VNode} from '@stencil/core';
+import {AnyBindings} from '../components/common/interface/bindings';
+
+export interface LocalizedStringProps {
+  bindings: AnyBindings;
+  key: string;
+  params: Record<string, VNode | string>;
+}
+
+export const LocalizedString: FunctionalComponent<LocalizedStringProps> = ({
```

Comment:
This component inserts JSX into a localized string without the need for `innerHTML` by returning an array of `VNode | string`.